src/style.css

```
@@ -1,3 +1,71 @@
+:root {
+  --main-color:  #00ACDC;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  width: 100%;
+  background-color: #fff;
+  box-sizing: border-box;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  display: block;
+  box-sizing: border-box;
+}
+
+.nav-list {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  display: flex;
+  gap: 20px;
+  box-sizing: border-box;
+}
+
+.nav-item {
+  margin: 0;
+}
+
+.nav-link {
+  display: inline-block;
+  line-height: 60px;
+  text-decoration: none;
+  font-size: 12px;
+  color: #000;
+  text-transform: uppercase;
+  text-align: center;
+  position: relative;
+  font-style: normal;
+  font-weight: 500;
+  box-sizing: border-box;
+}
+
+.nav-link:hover {
+  color: var(--main-color);
+}
+
+.is-active {
+  color: var(--main-color);
+}
+
+.is-active::after {
+  content: '';
+  display: block;
+  width: 100%;
+  height: 4px;
+  border-radius: 8px;
+  background-color: #00ACDC;
```

```suggestion
  background-color: var(--main-color);
```
forgot about this one :)

---

@TarasHoliuk oh, it was help from chat with mentor, his help. because I had color: var(--main-color); and it not worked. He saw me code and said to change it on background-color: #00ACDC. 

Okey, I change on  var(--main-color) after your comment =)